components/config-mgmt-service/params/parameters.go

```diff
@@ -137,7 +137,7 @@ func ConvertParamToActionBackend(parameter string) string {
 	switch parameter {
 	case "organization":
 		return backend.ActionOrganization
-	case "source_fqdn":
+	case "source_fqdn", "chef_server":
```

Comment:
im curious -- why not also include "service_hostname" here? is that not another alias?

Reply:
The "service_hostname" term is the real value that we use to filter on. The `backend.ActionSourceFQDN` equals "service_hostname". 
If we do want to add that term as part of our API we could add it here. That term seemed to me to be internal use only. Maybe I should remove it from the unit test :thinking:

Reply:
👍  i just didn't know what the term was